iam.tf

```diff
@@ -54,11 +54,12 @@ data "aws_iam_policy_document" "role" {
 }
 
 module "role_label" {
-  source     = "git::https://github.com/cloudposse/terraform-null-label.git?ref=tags/0.2.1"
-  namespace  = "${var.namespace}"
-  stage      = "${var.stage}"
-  name       = "${var.name}"
-  delimiter  = "${var.delimiter}"
+  source    = "git::https://github.com/cloudposse/terraform-null-label.git?ref=tags/0.2.1"
+  namespace = "${var.namespace}"
+  stage     = "${terraform.workspace}"
+  name      = "${var.name}"
+  delimiter = "${var.delimiter}"
+
   attributes = ["${compact(concat(var.attributes, list("role")))}"]
   tags       = "${var.tags}"
 }
```

Comment:
use variable instead of hardcode `"arn:aws:iam::aws:policy/service-role/AWSDataPipelineRole"`

```
resource "aws_iam_role_policy_attachment" "role" {
  role       = "${aws_iam_role.role.name}"
  policy_arn = "arn:aws:iam::aws:policy/service-role/AWSDataPipelineRole"
}
```